lightning/src/chain/channelmonitor.rs

```diff
@@ -2686,6 +2738,17 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitorImpl<Signer> {
 		ret
 	}
 
+	pub(crate) fn initial_counterparty_commitment_tx(&mut self) -> Option<CommitmentTransaction> {
+		let (their_per_commitment_point, feerate_per_kw, to_broadcaster_value,
+			to_countersignatory_value) = self.initial_counterparty_commitment_info.take()?;
```

Comment:
We probably don't need to, but I couldn't really think of another time when a user would want to call this besides on the first persist, and thought if we took here we wouldn't have to keep around and persist extra bytes (although it's not really much) - could see it either way, it might just be a simpler API to not have to think about whether it's been called before though?